db/schema.rb

```diff
@@ -57,20 +57,37 @@
     t.text "other_language_details"
     t.date "date_of_birth"
     t.text "further_information"
-    t.datetime "submitted_at"
     t.string "phone_number"
     t.string "address_line1"
     t.string "address_line2"
     t.string "address_line3"
     t.string "address_line4"
     t.string "country"
     t.string "postcode"
+    t.datetime "submitted_at"
```

Comment:
Should this be moving?

Reply:
I ran into this on #374 too — I think it's an artefact of running migrations out of order after merges, and it will shake out with the next schema dump.

Reply:
Also ran into it on https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/386